agent/replacecall/src/main/resources/default-load-exclusions.txt

```diff
@@ -0,0 +1,21 @@
+// This file contains default package exclusions for the replacecall agent.
```

Comment:
Why is this file named "load-exclusions.txt" rather than something like "replacement-exclusions.txt" or "replacecall-exclusions.txt"?  "load" is not very mnemonic.  Same comment for the file used in system tests.